maven-install.sh

```diff
@@ -0,0 +1,34 @@
+#group is overiden to support forked repositories
+
+function print_usage(){
+  echo "maven-install.sh --version VERSION [--group GROUP]"
+}
+
+for i in "$@"
+do
+  case "$1" in
+    --version)
+      VERSION="$2"
+      shift
+      ;;
+    --group)
+      GROUP="$2"
+      shift
+      ;;
+    --help)
+      print_usage
+      exit 0
+      ;;
+    *)
+      ;;
+  esac
+  shift
+done
+
+if [ -z "$VERSION" ]; then
+  print_usage
+  exit
+fi
+echo VERSION=$VERSION
+
+./gradlew install -Dorg.gradle.parallel=false -PuseHadoop2 -Pversion=0.6-SNAPSHOT -Pgroup=gobblin
```

Comment:
redone ... sorry :(
Hard end of week ...


On Fri, Nov 20, 2015 at 7:44 PM, Yinan Li <notifications@github.com> wrote:

> In maven-install.sh
> <https://github.com/linkedin/gobblin/pull/475#discussion_r45502533>:
>
> > +      print_usage
> > +      exit 0
> > +      ;;
> > +    *)
> > +      ;;
> > +  esac
> > +  shift
> > +done
> > +
> > +if [ -z "$VERSION" ]; then
> > +  print_usage
> > +  exit
> > +fi
> > +echo VERSION=$VERSION
> > +
> > +./gradlew install -Dorg.gradle.parallel=false -PuseHadoop2 -Pversion=0.6-SNAPSHOT -Pgroup=gobblin
>
> Seems you are not using the arguments here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linkedin/gobblin/pull/475/files#r45502533>.
>



-- 
"Computers are useless. They can only give you answers."
- Pablo Picasso -